app/src/main/java/com/mathroule/demoapp/MainActivity.java

```
@@ -28,6 +30,22 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            ((Button) findViewById(R.id.buton)).setText(text);
+
+            if (text == "something") {
+                ((Button) findViewById(R.id.buton)).setText("something");
+            }
+        } catch (Exception e) {
```

<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/602938afe0dadcf5133abec0fe37a99e44f2361a/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L41"></span>Empty catch block.</td>
    </tr>
  </tbody>
</table>